core/modules/replica/ConfigurationMySQL.h

```diff
@@ -98,174 +89,240 @@ class ConfigurationMySQL : public ConfigurationBase {
 
     std::string configUrl(bool showPassword=false) const final;
 
-    void setRequestBufferSizeBytes(size_t val) final {
+    void setRequestBufferSizeBytes(size_t val,
+                                   bool updatePersistentState) final {
         _set(_requestBufferSizeBytes,
              "common",
              "request_buf_size_bytes",
-             val);
+             val,
+             false,
```

Comment:
a comment expressing the use/intent of the bool var might be useful. and so on below.

Reply:
Good point! I will fix it.